### DISCUSSION TITLE
Reset User Password

### DISCUSSION THREAD

**Discussion by darthrater78** [advplyr/audiobookshelf#2355] [General]
I need to reset one of my user's passwords but I can't seem to find it in the GUI. My search showed resetting the root via json. But this is just a standard user.

**Comment by advplyr** [comment]
Press edit on a user
![image](https://github.com/advplyr/audiobookshelf/assets/67830747/2d77fd9a-73fd-4a3a-b3ae-a5b8206f4d82)

**Comment by darthrater78** [comment]
Wow, I feel silly. At least this will be searchable for future questioners. Thanks!
